app/views/application/confirmation.njk

```diff
@@ -9,20 +16,20 @@
 
   {{ govukPanel({
     classes: "govuk-!-margin-bottom-8",
-    titleText: "Application successfully submitted",
+    titleText: title,
     html: panelText
   }) }}
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <h2 class="govuk-heading-m">What happens next</h2>
-      <p class="govuk-body">You will receive an email from the Apply for postgraduate teacher training service to confirm the successful submission of your application.</p>
+      <p class="govuk-body">The Apply for teacher training service will send you an email to confirm the successful submission of your application.</p>
       <h2 class="govuk-heading-m">References</h2>
       <p class="govuk-body"> We will also contact your referees to ask for your references. When we receive them, we will send them to your training provider. Your provider then has 40 working days to respond to your application.</p>
       <h2 class="govuk-heading-m">Interview</h2>
       <p class="govuk-body">If your training provider decides to offer you an interview, they will get in touch directly via email to organise a date and give you all the information you need.</p>
       <h2 class="govuk-heading-m">Track, edit or withdraw</h2>
-      <p class="govuk-body">To make changes to a submitted application, return to the <a href="/applications">applications dashboard.</a> </p>
+      <p class="govuk-body">You have 5 working days to make changes to a submitted application. To edit, return to your <a href="{{ nextHref }}">Application dashboard</a>.</p>
```

Comment:
Can we make this bit specific to the submitted application, eg "You have until 10 October to make changes to your application"?